CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.52.0
+* Enhancement: Support for ECS EXTERNAL launch type (ECS Anywhere)
```

Comment:
could we include changelog from https://github.com/aws/amazon-ecs-agent/pull/2837:
```
Enhancement - Add support for ECS agent to acknowledge server heartbeat messages
```